pkg/db/task.go

```diff
@@ -0,0 +1,157 @@
+package db
+
+import (
+	"database/sql"
+	"fmt"
+	"strconv"
+	"time"
+)
+
+type Task struct {
+	ID      string `json:"id"`
+	Date    string `json:"date"`
+	Title   string `json:"title"`
+	Comment string `json:"comment"`
+	Repeat  string `json:"repeat"`
+}
+
+func AddTask(task *Task) (int64, error) {
+	query := `INSERT INTO scheduler (date, title, comment, repeat)
+	          VALUES (?, ?, ?, ?)`
+	res, err := DB.Exec(query, task.Date, task.Title, task.Comment, task.Repeat)
+	if err != nil {
+		return 0, err
+	}
+	return res.LastInsertId()
+}
+
+func GetTask(id string) (*Task, error) {
+	query := `SELECT id, date, title, comment, repeat
+	FROM scheduler
+	WHERE id = ?`
+
+	row := DB.QueryRow(query, id)
+	t := &Task{}
+	err := row.Scan(&t.ID, &t.Date, &t.Title, &t.Comment, &t.Repeat)
+	if err != nil {
+		if err == sql.ErrNoRows {
+			return nil, fmt.Errorf("task %s not found", id)
+		}
+		return nil, err
+	}
+	return t, nil
+}
+
+func DeleteTask(id string) error {
+	query := `DELETE FROM scheduler WHERE id = ?`
+
+	i, err := strconv.ParseInt(id, 10, 64)
+	if err != nil {
+		return fmt.Errorf("invalid id: %w", err)
+	}
+
+	res, err := DB.Exec(query, i)
+	if err != nil {
+		return err
+	}
+	count, err := res.RowsAffected()
+	if err != nil {
+		return err
+	}
+
+	if count == 0 {
+		return fmt.Errorf("incorrect id for deleting task")
+	}
+	return nil
+}
+
+func UpdateTask(task *Task) error {
+	query := `UPDATE scheduler
+	SET date = ?, title = ?, comment = ?, repeat = ?
+	WHERE id = ?`
+
+	res, err := DB.Exec(query, task.Date, task.Title, task.Comment, task.Repeat, task.ID)
+	if err != nil {
+		return err
+	}
+
+	count, err := res.RowsAffected()
+	if err != nil {
+		return err
```

Comment:
Лучше всегда оборачивать ошибку
```suggestion
		return fmt.Errorf("invalid id for updating task: %w", err)
```